packages/itwinui-css/src/bridge.scss

```diff
@@ -135,6 +144,12 @@
   /* Button */
 
   .iui-button {
+    // StrataKit button background color is not iTwinUI's --iui-color-background global mapping
+    &:not(.iui-input-flex-container-icon):not([data-iui-variant]) {
```

Comment:
Why is the `:not(.iui-input-flex-container-icon)` part necessary? InputFlexContainer should only use borderless buttons which will already be covered by `:not([data-iui-variant])`.

Reply:
Also, could you wrap these (and other) nested selectors with `:where()` to keep the specificity flat? Specificity should remain flat for modifiers on the same element, and only increase when selecting new elements.